Cargo.toml

```diff
@@ -15,6 +15,10 @@ license = "CDDL-1.0"
 
 exclude = ["/tests/**", "/flamegraph/**", "/*.perf"]
 
+[profile.release]
+strip = true   # To use flamegraph on inferno binaries, comment this line
+# debug = true # and uncomment this line.
+
```

Comment:
I moved this to make it the second section, so that people don't miss it, and added these comments just in case someone tries to use these binaries with `flamegraph`.